configure/operator.md

```diff
@@ -83,16 +98,18 @@ The ``backup`` section in the [deploy/cr.yaml](https://github.com/percona/percon
 
 | Key                            | Value Type | Example   | Description |
 |--------------------------------|------------|-----------|-------------|
-|image                           | string     | `perconalab/backupjob-openshift:0.2.0` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
+|image                           | string     | `perconalab/percona-xtradb-cluster-operator:0.4.0-backup` | Percona XtraDB Cluster docker image to use for the backup functionality                                                                       |
 |imagePullSecrets.name           | string     | `private-registry-credentials`  | [Kubernetes imagePullSecret](https://kubernetes.io/docs/concepts/configuration/secret/#using-imagepullsecrets) for the specified docker image |
 |storages.type                   | string     | `s3`      | Type of the cloud storage to be used for backups. Currently only `s3` and `filesystem` types are supported |
 |storages.s3.credentialsSecret   | string     | `my-cluster-name-backup-s3`| [Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/) for backups. It should contain `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` keys. |
 |storages.s3.bucket              | string     |           | The [Amazon S3 bucket](https://docs.aws.amazon.com/en_us/AmazonS3/latest/dev/UsingBucket.html) name for backups     |
 |storages.s3.region              | string     |`us-east-1`| The [AWS region](https://docs.aws.amazon.com/en_us/general/latest/gr/rande.html) to use. Please note **this option is mandatory** not only for Amazon S3, but for all S3-compatible storages.|
-|storages.s3.endpointUrl         | string     |           | The endpoint URL of the S3-compatible storage to be used (not needed for the original Amazon S3 cloud)   |
+|storage.s3.endpointUrl | string|           | The endpoint URL of the S3-compatible storage to be used (not needed for the original Amazon S3 cloud) |
+|storage.type                    | string    | `filesystem` | persistent volume type |
```

Comment:
duplicate, see line 88, 103

Reply:
One is for the persistentvolume - added text to make the distinction.